src/splitInteger.js

```diff
@@ -4,6 +4,8 @@
  *
  * @returns {number[]}
  */
+'use strict';
```

Comment:
The 'use strict' directive is not needed in ES6 modules. If this is an ES6 module, consider removing the line. Otherwise, it's fine to keep it. 😊